### ISSUE TITLE
Cannot find module './configs/all' with version 1.9.0

### ISSUE THREAD

**Issue by svanherk** [43081j/eslint-plugin-lit#177]
Starting with version 1.9.0 we're getting this error:
```
Error: Failed to load plugin 'lit' declared in '.eslintrc.json » <our_shared_eslint_config_library>/<our_shared_eslint_config>': Cannot find module './configs/all'
Require stack:
- .../node_modules/eslint-plugin-lit/lib/index.js
- .../node_modules/@eslint/eslintrc/dist/eslintrc.cjs
```

It looks like this is a change in how later versions of `npm` do the packing.  It packs as expected locally when using npm `8.19.4` (on Node 16) , but for npm `9.6.7` (on Node 18) and npm `9.8.0` (on Node 20) it excludes the `lib/configs/recommended.*`, `lib/config/all.*`, and `lib/rules/*` files.

**Comment by 43081j** [comment]
yep, you are indeed correct

it is caused by npm unknowingly shipping a breaking change in how globs work (see npm/npm-packlist#152)

have published 1.9.1

**Comment by svanherk** [comment]
Awesome, thank you! Error is gone with the new version.
